spec/models/company_spec.rb

```diff
@@ -16,7 +16,9 @@
         attribute = :name
         value = 'foo'
         return_attr = 'name'
+
         actual = described_class.get_record_val_by(attribute, value, return_attr)
+
```

Comment:
Actually, the entire test context for `.get_record_val_by` is incomplete and I'm going to test it further while making use of this idea.